src/pynwb/ndx_patterned_ogen/patterned_ogen.py

```diff
@@ -524,3 +614,54 @@ def add_interval(self, **kwargs):
         Add a stimulation parameters for a specific stimulus onset.
         """
         super(PatternedOptogeneticStimulusTable, self).add_interval(**kwargs)
+
+        if isinstance(kwargs["power"], (list, np.ndarray, tuple)):
```

Comment:
Or I'm guessing this is towards the goal of having a more informative error message...

If you forbid iterable types in the docval, what is the error message when you try to call this method using `power` as if you meant `power_per_rois`?

Reply:
`TypeError: PatternedOptogeneticStimulusTable.add_interval: incorrect type for 'power' (got 'list', expected 'int or float')`
It's informative, but I wanted to suggest to use power_per_roi instead. Do you think it would be better to forbid the type iterable for power, frequency, and pulse_width and just specify to use .._per_roi in the documentation?